packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -161,3 +171,28 @@ export class UpdateAadAppDriver implements StepDriver {
     };
   }
 }
+
+export async function getQuestionForDeployAadManifest(
```

Comment:
Personally, I would prefer to put these 2 functions to another place, because the driver itself does not care about the questions.

Reply:
mvoe to questions.ts